<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseModsWindow</class>
 <widget class="QWidget" name="ChooseModsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select active mods</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListView" name="listViewEnabled"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="setEnabledButton">
           <property name="text">
            <string>Set enabled</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="enableAllButton">
           <property name="text">
            <string>Enable all</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="installModButton">
         <property name="text">
          <string>Install a mod</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteModButton">
         <property name="text">
          <string>Delete mod</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="disableAllButton">
           <property name="text">
            <string>Disable all</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="setDisabledButton">
           <property name="text">
            <string>Set disabled</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="listViewDisabled"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="resetConfigButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Reset from config</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="validateButton">
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
